audonnx/core/api.py

```diff
@@ -34,6 +40,9 @@ def load(
             In legacy mode path to model ONNX file
         labels_file: YAML file with labels
         transform_file: YAML file with transformation
+        device_or_providers: set device
+            (``'cpu'``, ``'cuda'``, or ``'cuda:<id>'``)
+            or a list of providers_
```

Comment:
We also allow for a single provider to be provided as string.

Reply:
Maybe: 

![image](https://user-images.githubusercontent.com/10383417/152960579-4c0360bc-1192-4b27-8c49-53b96ad738bd.png)

Reply:
Maybe we should limit provider always to a sequence?
Because now you can use `device='CUDAExecutionProvider'`, but not `device=('CUDAExecutionProvider', {...})`, or will this work?

Reply:
Yes, that's better.